### ISSUE TITLE
URL's without / pull in every branch of userhelp into the homepage

### ISSUE THREAD

**Issue by sachajudd** [silverstripe/userhelp.silverstripe.org#64]
URL's that don't end in / for example userhelp.silverstripe.org/en/3.5 pull in every branch of userhelp content onto the homepage which ends up looking broken. Possibly could be a bug in our developer docs also. 

![image 1](https://user-images.githubusercontent.com/24258161/41317441-4db25a54-6ee9-11e8-9df7-d3c5d0409a8f.png)

**Comment by robbieaverill** [comment]
Yeah this is a problem in the docs.silverstripe.org site as well. It's showing the main navigation items on the left, but for every available version while not distinguishing them visually with groups or titles. This is because docsviewer can't decide which version you were or wanted to be looking at, so it shows all of them instead. Needs some kind of fix though...

**Comment by robbieaverill** [comment]
I think there's two fixes here - first of all `en/3` should resolve correctly, the same as `en/3/` would. The second fix is that we should only show the navigation for the default/current/stable version in the 404 page rather than all navigation items for all versions.
